NUEntity.js

```diff
@@ -179,7 +179,7 @@ export default class NUEntity extends NUObject {
         const entity = this;
         entity._validators.forEach((validator, attributeName) => {
             const attrObj = this.constructor.attributeDescriptors[attributeName];
-            const validationError = validator.validate.call(entity, attrObj, formValues);
+            const validationError = validator.validate(attrObj, formValues);
```

Comment:
pass in the entity to the validate as the first argument.`validator.validate(entity, attrObj, formValues);`